agent/src/index.js

```diff
@@ -21,7 +21,7 @@ let LOG_PATH = process.env.LOG_PATH || '/logs/ceramic/'
 if (!LOG_PATH.endsWith('/')) LOG_PATH += '/'
 
 const CERAMIC_NETWORK = process.env.CERAMIC_NETWORK || 'dev-unstable'
-const { IPFS_API_URL } = process.env
+const { IPFS_API_URL } = ''  // if set, we get errors
```

Comment:
This is curious. Why are we getting errors here? Seems that making this an empty string would break functionality if the agent isn't able to connect to an ipfs api.

Let's find out what the issue is.

Reply:
yes, this is outdated - we should not be cahnging the javascript agent at all, removed this change